app/views/sidebar/member.js

```diff
@@ -50,8 +50,15 @@ view.render(() =>
     h3('.h3', 'Teams'),
     p(state.get('member').team),
     p(state.get('member').productTeam),
-    h3('.h3', 'Reports to'),
-    p(state.get('member').reportsTo)
+    div(state.get('member').reportsTo
+      ? div(
```

Comment:
I believe in React you can use an array for a collection of elements instead of having to wrap in a div. That said, not sure if I supported that in Veact. Either way, it isn't that big of a deal to wrap this in another div, so 👍